hologram_dashboard/fetcher/flags.cc

```diff
@@ -16,11 +16,26 @@
 
 #include "flags.h"
 
-ABSL_FLAG(std::string, chipper_batch_job_cell, "", 
-    "The job running cell of Chipper Batch.");
-ABSL_FLAG(std::string, chipper_gdpr_batch_job_cell, "", 
-    "The job running cell of Chipper GDPR pipeline.");
 ABSL_FLAG(std::string, hologram_source_config_file_path, "", 
     "Specifies where the config file can be located.");
 ABSL_FLAG(absl::Time, hologram_data_availability_check_time, absl::Now(), 
-    "The time for which you are interested in hologram data availability. By default, it uses current time.");
\ No newline at end of file
+    "The time for which you are interested in hologram data availability. By default, it uses current time.");
+ABSL_FLAG(std::vector<std::string>, chipper_required_sources, 
+    std::vector<std::string>(), "required");
+ABSL_FLAG(std::vector<std::string>, chipper_gdpr_required_sources);
+ABSL_FLAG(std::string, chipper_job_start_root, "",
+    "Specifies the root directory for CHIPPER job start time.");
+ABSL_FLAG(std::string, chipper_gdpr_job_start_root, ""
+    "Specifies the root directory for CHIPPER_GDPR job start time.");
+ABSL_FLAG(std::string, android_spam_done_file_root, ""
+    "Specifies the root directory for data injection done files for HOLOGRAM_ANDROID_SPAM_ABUSE.");
+ABSL_FLAG(std::string, prime_meridian_done_file_root, ""
+    "Specifies the root directory for data injection done files for HOLOGRAM_PRIME_MERIDIAN.");
+ABSL_FLAG(std::string, default_done_file_root, ""
+    "Specifies the root directory for data injection done files for HOLOGRAM_DEFAULT.");
+ABSL_FLAG(std::string, spam_done_file_root, ""
+    "Specifies the root directory for data injection done files for HOLOGRAM_SPAM.");
+ABSL_FLAG(std::string, user_attribute_done_file_root, ""
+    "Specifies the root directory for data injection done files for HOLOGRAM_USER_ATTRIBUTE.");
+ABSL_FLAG(std::string, user_experiment_done_file_root, "",
+    "Specifies the root directory for data injection done files for HOLOGRAM_USER_EXPERIMENT_ID_AND_TEST_CODE.");
```

Comment:
I am a bit confused about all these flags. 

So to make sure I understand, we have two types of DONE files, one is for data source ingestion, the other is for kvick servers to update data.
For data sources, the path is 
/cns/vr-d/home/hologram_default/2020/06/15/134567/DONE
/cns/vr-d/home/hologram_SPAM/2020/06/15/137890/DONE
...
For kvick servers, the path is 
/cns/vr-d/home/coordinator_update/2020/06/15/134566/DONE
/cns/vr-d/home/lookup_server_update/2020/06/15/134568/DONE
...
You probably don't need one flag per system DONE file root? +Sarah. 
We can discuss more offline if that's easier.

Reply:
I have synced with Alex, I asked him to add these 6 flags. We can sync in meeting if it confuses you.